be/src/olap/page_cache.cpp

```diff
@@ -42,11 +42,18 @@ bool StoragePageCache::lookup(const CacheKey& key, PageCacheHandle* handle) {
     return true;
 }
 
-void StoragePageCache::insert(const CacheKey& key, const Slice& data, PageCacheHandle* handle) {
+void StoragePageCache::insert(const CacheKey& key, const Slice& data, PageCacheHandle* handle,
+                              bool in_memory) {
```

Comment:
I think is_durable is a better name?

Reply:
No.
`durable` is for `LRUCache`,
`in_memory` is for storage level.